doc/contributing/documentation_guide.md

```diff
@@ -137,6 +137,15 @@ or [list](rdoc-ref:RDoc::Markup@Simple+Lists)
 should be preceded by and followed by a blank line.
 This is unnecessary for the HTML output, but helps in the `ri` output.
 
+### \Method Names
+
+For a method name in text:
+
+- When the class name is included, use a dot or a hash mark:
```

Comment:
The current wording makes it sound a bit like these two syntax are mean the same and are interchangable. 

```suggestion
- When the class name is included, use a dot for class methods or a hash mark for instance methods:
```

Reply:
Fixed.